packages/neo-one-client-common/src/models/transaction/FeelessTransactionModel.ts

```diff
@@ -84,7 +82,7 @@ export class FeelessTransactionModel<
     this.validUntilBlock = validUntilBlock;
     this.script = script;
     const hashIn = hash;
-    this.hashInternal = hashIn === undefined ? utils.lazy(() => crypto.hash256(this.message)) : () => hashIn;
+    this.hashInternal = hashIn === undefined ? (magic) => crypto.hash256(this.getMessage(magic)) : () => hashIn;
```

Comment:
I think this is way worse for how stuff will flow now, especially when looking at the `clone` method.

We should either have a `this.magic` property that gets set in the constructor or from the deserializeWireOptions context, or have a utils function that takes in a serializable. This constant passing through of magic is no bueno imo.